CONTRIBUTING.md

```diff
@@ -1,363 +1,24 @@
-# Contributing
+# Bidra
 
-Forbedring av dokumentasjon, bugfixes, åpning av issues med spørsmål, bugrapporter, eller feature-ønsker - alle kan
-bidra, og ingen bidrag er for små! :tada:
+Så kjekt at du har lyst å bidra til at koden vår blir enda bedre! I dette dokumentet har vi satt sammen en kortfattet guide for å hjelpe nye såvel som drevne utviklere i gang. Vi oppfordrer samtidig alle til å holde en hyggelig og inkluderende tone. Alle har noe de kan bidra med, og alt er åpent for diskusjon. Mer om dette i vår "[Code of Conduct](https://github.com/SpareBank1/designsystem/blob/develop/CODE_OF_CONDUCT.md)".
 
-I dette dokumentet vil du finne alt du skulle trenge av informasjon for å hjelpe til med å gjøre frontend-biblioteket
-vårt enda bedre. Sitter du igjen med ubesvarte spørsmål etter å ha lest dette dokumentet, er terskelen lav for å åpne et
-issue her på Github med `question`-labelen. Om du heller vil kan du også spørre på den interne HipChat-kanalen
-"Designsystem".
+## Funnet en feil, lyst å komme med innspill, eller andre tilbakemeldinger?
 
--   [Hvordan bidrar jeg?](#hvordan-bidrar-jeg)
-    -   [Som designer](#som-designer)
-        -   [Github-konto og varsler](#github-konto-og-varsler)
-    -   [Som utvikler](#som-utvikler)
-        -   [Github-konto og arbeidsflyt](#github-konto-og-arbeidsflyt)
-        -   [Utviklingsmiljø](#utviklingsmiljø)
-        -   [Kodestandard](#kodestandard)
-        -   [Formatering](#formatering)
-        -   [LESS](#less)
-        -   [React](#react)
-        -   [Testdekning](#testdekning)
-        -   [Dokumentasjon](#dokumentasjon)
-        -   [Commit-meldinger](#commit-meldinger)
-        -   [Pull requests](#pull-requests)
--   [Lage en ny FFE-komponent](#lage-en-ny-ffe-komponent)
--   [Publisering](#publisering)
--   [Har du fortsatt sporsmål?](#har-du-fortsatt-spørsmål)
+For å kunne jobbe mest mulig effektivt ønsker vi at forslag kommer inn via [Issues](https://github.com/SpareBank1/designsystem/issues). Forslag til forbedringer, diskusjonstråder og spørsmål er også hjertelig velkomne her. Når du oppretter et nytt "issue" vil det dukke opp en mal vi anbefaler at du følger som best kan. Mer informasjon om hvordan du lager issue finner du [her](https://github.com/SpareBank1/designsystem/wiki/Bidra:-Fra-issue-til-pull-request#issues).
 
-## Hvordan bidrar jeg?
+## Vi bruker [GitHub Flow](<(https://guides.github.com/introduction/flow/)>), og foretrekker endringer som Pull Requets.
```

Comment:
Et øyeblikk der trodde jeg det var snakk om git-flow fra Atlassian. Veldig glad for å se at det ikke er tilfelle :)